geode-core/src/distributedTest/java/org/apache/geode/internal/cache/versions/TombstoneDUnitTest.java

```diff
@@ -61,6 +61,7 @@
   private static Cache cache;
   private static Region<String, String> region;
   final String REGION_NAME = "TestRegion";
+  final int EXPIRY_TIME=600000;
```

Comment:
I think it would be better to directly reference the default tombstone timeout using `REPLICATE_TOMBSTONE_TIMEOUT_DEFAULT` since if it changes, then this value will no longer be correct.

Reply:
Done.